wrapper/src/test/java/integration/util/AuroraTestUtility.java

```diff
@@ -923,4 +928,33 @@ public void addAuroraAwsIamUser(
       }
     }
   }
+
+  public List<String> getEngineVersions(String engine) {
+    final List<String> res = new ArrayList<>();
+    final DescribeDbEngineVersionsResponse versions = rdsClient.describeDBEngineVersions(
+        DescribeDbEngineVersionsRequest.builder().engine(engine).build()
+    );
+    for (DBEngineVersion version : versions.dbEngineVersions()) {
+      res.add(version.engineVersion());
+    }
+    return res;
+  }
+
+  public String getLatestVersion(String engine) {
+    return getEngineVersions(engine)
+        .stream()
+        .sorted()
+        .reduce((first, last) -> last)
```

Comment:
Can we use sorted() to sort collection and then return first() or last()?
https://www.baeldung.com/java-8-sort-lambda